zerver/tests/test_scim.py

```diff
@@ -182,12 +182,13 @@ def test_get_all_with_pagination(self) -> None:
             "totalResults": UserProfile.objects.filter(realm=realm, is_bot=False).count(),
             "itemsPerPage": 50,
             "startIndex": 1,
-            "Resources": [],
+            "Resources": [
+                self.generate_user_schema(user_profile)
+                for user_profile in UserProfile.objects.filter(realm=realm, is_bot=False).order_by(
+                    "id"
+                )
+            ],
         }
-        for user_profile in UserProfile.objects.filter(realm=realm, is_bot=False).order_by("id"):
-            user_schema = self.generate_user_schema(user_profile)
-            assert isinstance(expected_response_schema["Resources"], list)
-            expected_response_schema["Resources"].append(user_schema)
```

Comment:
The `test_scim` changes should be rebased back to “test: Add isinstance check for dictionary values”, not “scim: Add SCIMConfigDict”.